### PR TITLE
final version of post

### PR DESCRIPTION
**Pull request by robintrmbtt**

### CODE REVIEW

File: collections/_posts/2024-03-20-ImageBind.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+The authors use several dataset to couple audio, depth maps, thermal data and IMU to image (or video). Each dataset is split into train/test to evaluate zero-shot classification or retrieval abilities.
+
+<div style="text-align:center">
+<img src="/collections/images/imagebind/databases.jpg" width=500></div>
+<p style="text-align: center;font-style:italic">Figure 2. Overview of datasets used.</p>
+
+## Emergent zero-shot classification
+
+Zero-shot is a term used for model trained with a modality X (image or audio for instance) paired with text. The class is determined using simple standard prompts, such as "An image of a {class}", where {class} is replaced by every possible class. The class attributed to a sample is the one for with the associated text prompt has the highest similarity with the sample.
```
Comment (ThierryJudge) [inline collections/_posts/2024-03-20-ImageBind.md:73]: "Zero-shot is a term used for model trained with a modality X (image or audio for instance) paired with text". I do not understand what you mean here. 

My understanding would be 
" Zero-shot is a term used for a model trained with a modality pair X-Y (image or audio for instance) paired with text and applied to a modality pair Y-Z during inference."

File: collections/_posts/2024-03-20-ImageBind.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+* **Depth and Thermal data :** They treat both as one-channel images and use ViT to encoder them. 
+
+* **IMU :** The 6-channel signal acquisition is cut into 5-second clips, giving 2K time steps IMU. They are projected using 1D convolution with kernel size of 8 and then encoded using a Transformer.
+
+&nbsp;
+
+# Experiments
+
+## Datasets
```
Comment (ThierryJudge) [inline collections/_posts/2024-03-20-ImageBind.md:63]: Is the model trained once on the combination of all the datasets? If so, what is the size of the full training dataset?

**Review by ThierryJudge — CHANGES_REQUESTED** [review]
Good review. I just have a few questions and pointed out some typos.

File: collections/_posts/2024-03-20-ImageBind.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+* The authors propose ImageBind, an approach to learn a joint embedding space across six different modalities.
+* It is trained in a self-supervised fashion only with image-paired data, but can successfully bind together all modalities together.
+* It is evaluate on many downstream applications, such a zero- and few-shot classification, cross-modal retrieval and generation.
+
+&nbsp;
+
+# Context and objectives
+
+Many recent have worked on align image features with other modalities, such as text or audio. Such networks, like the popular CLIP, They exhibit strong zero-shot performances on downstream tasks and provide powerful feature extractor.However, they are limited to pairs of modalities for training and resulting downstream tasks.
```
[the remark this answers is not in the document]
Comment (robintrmbtt) [reply collections/_posts/2024-03-20-ImageBind.md]: yes, and i've reworded it

File: collections/_posts/2024-03-20-ImageBind.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+&nbsp;
+
+# ImageBind
+
+<div style="text-align:center">
+<img src="/collections/images/imagebind/overview.jpg" width=800></div>
+<p style="text-align: center;font-style:italic">Figure 1. Overview of ImageBind.</p>
+
+ImageBind uses pairs of modalities $$(\mathcal{I}, \mathcal{M})$$ where $$\mathcal{I}$$ represents images and $$M$$ another modality. In their work, $$\mathcal{M}$$ can be text, audio, depth sensor data, thermal data, and Inertial Measurment Unit (IMU, times-series data from accelerometers and gyroscopes). Note that image can replaced by image by inflating patch projection layers, thus both are treated with the same encoder.
```
[the remark this answers is not in the document]
Comment (robintrmbtt) [reply collections/_posts/2024-03-20-ImageBind.md:35]: it was supposed to be image can be replaced by video, but i've rephrased it

File: collections/_posts/2024-03-20-ImageBind.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Each modality $$M$$ has its own dedicated encoder (see below for details) and an associated projector $$f_\mathcal{M}$$, so that every modality embedding ends up in a space of common dimension.
```
[the remark this answers is not in the document]
Comment (robintrmbtt) [reply collections/_posts/2024-03-20-ImageBind.md:37]: yes

File: collections/_posts/2024-03-20-ImageBind.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+$$
+\mathcal{L}_{\mathcal{I},\mathcal{M}} = - \log \frac{\exp(\textbf{q}_i^T \textbf{k}_i / \tau)}{\exp(\textbf{q}_i^T \textbf{k}_i / \tau) + \sum_{i\neq j} \exp(\textbf{q}_i^T \textbf{k}_j/\tau)}
+$$
+
+&nbsp;
+
+## Modality encoders details
+
+* **Image and video :** Vision Transformer (ViT). For video , they use 2-frame clips, with a video patch of size $$2 \times 16 \times 16$$ ($$T\times H \times W$$). Encode's weights are inflated to work with spatiotemporal patches and at inference, features from successive 2-frame clips are aggregated. They leverage image-text supervision from large-scale data by using OpenCLIP's image encoder weights (ViT-H, 630M parameters). The weights are kept frozen.
```
[the remark this answers is not in the document]
Comment (robintrmbtt) [reply collections/_posts/2024-03-20-ImageBind.md:49]: done

File: collections/_posts/2024-03-20-ImageBind.md (hunk 1 above)
Comment (robintrmbtt) [reply collections/_posts/2024-03-20-ImageBind.md]: It was not clear ideed, i've reworded this paragraph

**Review by ThierryJudge — APPROVED** [review]

**Pull request merged**